Source/main.py

```diff
@@ -7,6 +11,16 @@
     # For each image in training set
         ##### Image Pre-processing #####
         ##### Feature Extraction #####
+    image_path = '../Data/Vehicules1024/00000001_ir.png'
+    image = imread(image_path )
+    image_window = image[99:199,99:199 ] # random block
```

Comment:
Can you move the test code into it's own function or module?  Since the HOG calculation is just a single function call I'm cool with it existing in the main function.